W-19615880 telemetry-exporter.adoc (#410)
mulesoft/docs-monitoring: modules/ROOT/pages/telemetry-exporter.adoc

** For Runtime Fabric deployments, see xref:runtime-fabric::use-anypoint-monitoring.adoc#enable-trace-data[Runtime Fabric].
* An endpoint for telemetry data that is accessible from across the internet (CIDR `0.0.0.0/0`) if your observability provider or your organization's endpoint requires an IP allowlist or firewall rule
+

> Consult your organization’s security guidelines when configuring open access rules for this endpoint. Telemetry Exporter generates its IP address dynamically, so allowlisting specific IPs, URLs, domains is not possible at this time.

Author: ```suggestion
Consult your organization’s security guidelines when configuring open access rules for this endpoint. Telemetry Exporter generates its Internet Protocol (IP) address dynamically, which prevents allowlisting specific IPs, URLs, domains.
```
